src/search-modal/__mocks__/search-result.json

```diff
@@ -365,6 +365,208 @@
         }
       },
       "facetStats": {}
+    },
+    {
+      "indexUid": "studio",
+      "hits": [
+        {
+          "display_name": "Collection 1",
+          "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque et mi ac nisi accumsan imperdiet vitae at odio. Vivamus tempor nec lorem eget lacinia. Vivamus efficitur lacus non dapibus porta. Nulla venenatis luctus nisi id posuere. Sed sollicitudin magna a sem ultrices accumsan. Praesent volutpat tortor vitae luctus rutrum. Integer.",
+          "id": 1,
+          "type": "collection",
+          "breadcrumbs": [
+            {
+              "display_name": "CS problems 2"
+            }
+          ],
+          "created": 1725534795.628254,
+          "modified": 1725534795.628266,
+          "context_key": "lib:OpenedX:CSPROB2",
```

Comment:
This mock is combining course results and library results which seems wrong. I would prefer to have a separate mock for libraries. You can base your work on the mock I made in https://github.com/openedx/frontend-app-course-authoring/pull/1263 - see `[src/library-authoring/__mocks__/library-search.json](https://github.com/openedx/frontend-app-course-authoring/pull/1263/files#diff-923384689ec224772d7555023d1eab8ae7a2c90e930727e972540b8c49534cb0)`

Reply:
Ohh, did not notice that. Thanks! rebased your work and updated the test.